HW6.py

```diff
@@ -0,0 +1,224 @@
+
+    def advertisement(self):
+        """Advertising of master. Skills always increase by two"""
+        print('Highly skilled master {0} from {1} company is looking for work!'
+              ' My skill is {2}. Prices:\nImport refrigerator: {3}\nDomestic '
+              'refrigerator: {4}'.format(self.name, self.company,
+                                         str(self.skill + 2),
+                                         str(self.skill * 150),
+                                         str(self.skill * 100)))
+
+    def try_to_fix(self, customer, refr):
+        """Master will try to fix refrigerator
+
+        Price formula: skill * 100 * сoeff
+        Netherlands refrigerators can be fixed by only masters with >=7 skill,
+        coeff is 1,5
+        Belarus refrigerators can be fixed only by masters with >=3 skill,
+        coeff is 1, and also help to improve some skills.
+        Payment is made by the client (Parent)
+        Сonsultation is free.
+        """
+        if refr.made_in == 'The Netherlands':
+            if self.skill >= 7:
+                refr.broken = False
+                customer.money -= self.skill * 100 * 1.5
+                return True
+        elif refr.made_in == 'Belarus':
+            if self.skill >= 3:
+                refr.broken = False
+                customer.money -= self.skill * 100 * 1
+                self.skill += 0.5
+                return True
+        return False
+
+    def study(self):
+        """Master can pay money and improve his skills"""
+        if self.money >= 100:
+            self.money -= 100
+            self.skill += 1
+        else:
+            print('{self.master} don\'t have enough money to study')
+
+
+class Product(object):
+    """Product is some food we can eat or put to refrigerator
+
+    It can be cold and freeze type.
+    Every product aso have size -oit shows how many cells this product need
+    """
+    in_cold_cam = None
+    in_freeze_cam = None
+
+    def __init__(self, name, pr_type='cold', size=1):
+        self.name = name
+        self.pr_type = pr_type
+        self.size = size
+
+
+class Refrigerator(object):
+    """Refrigerator is place where we can save products
+
+    New refrigerator always is empty and able to work
+    """
+    freeze_cells = 0
+    cold_cells = 0
+    broken = False
+
+    def __init__(self, name):
+        self.name = name
+
+    def display_info(self):
+        """Show information about refrigerator and free cells"""
+        print('Hi, I am {0} aka {1}\nNow it is {2} free cold cells'
+              ' and {3} free freeze cells '.format(type(self).__name__,
+                                                   self.name,
+                                                   self.cold_cells,
+                                                   self.freeze_cells))
+
+
+class RefrigeratorAtlant(Refrigerator):
+    """Belorussian refrigerator with small capacity"""
+    made_in = 'Belarus'
+    freeze_cam_capacity = 4
```

Comment:
i don't understand why it doesn't depend on model of fridge

Reply:
Fixed.
Now every company has own models row. And we can change model or create new using "create_or_update_model" class method